library/Xi/Netvisor/Netvisor.php

```diff
@@ -230,6 +230,24 @@ public function getSalesInvoice($id)
         );
     }
 
+    /**
+     * Get sales invoices by last time modified and invoices above Netvisor key
+     *
+     * @param dateTime $modifiedAfterDate
+     * @param int $invoiceId
+     * @return null|string
+     */
+    public function getSalesInvoices(\DateTime $modifiedAfterDate, int $invoiceId)
```

Comment:
This for our application purpose exactly, not a generic solution as package should have. Well examples are not too much better either 😆 

I now get the temptation about the array param... The basic problem is that those two params are currently required and invoiceId is poorly named. At least something like $aboveKey or $greaterThanKey etc.

But thinking ahead, I can see how confusing the param list will get over time if more filters are needed, and theyd  be in very illogical order. So I'd really prefer a class like SalesInvoiceFilter which has setters for filters and a method for outputting the array of filters for the request. But I leave that up to you, doesn't play much difference atm. if you think you have time for it, please do, but those slight changes are enough for acceptance.

Reply:
I think that filter class is nice. Added that.